script/tax_household_groups/copy_application_id_to_application_hbx_id.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# This script triggers the operation to copy the application id to the application hbx id for all the tax household groups.
+#
+# Command to trigger the script:
+#   CLIENT=me bundle exec rails runner script/tax_household_groups/copy_application_id_to_application_hbx_id.rb
+
+p '********** STARTING - Script to copy the application id to the application hbx id for eligible tax household groups. **********'
+
+# Calls the operation to copy the application id to the application hbx id for all the tax household groups.
+# If the job is successful, it logs the success message and instructions.
+# If the job fails, it logs the failure message.
+#
+# @return [void]
+result = ::Operations::Migrations::TaxHouseholdGroups::CopyApplicationIdToApplicationHbxId.new.call
```

Comment:
Just a thought-- should we bench mark the time to complete here?  I know it's helpful for Prod deployment to know how long the migration roughly takes.